slo-monitor/.gitignore

```diff
@@ -0,0 +1,24 @@
+build
```

Comment:
This file should probably be in top level directory (if we want this merged at all).